### ISSUE TITLE
Some page titles don't describe the page well enough

### ISSUE THREAD

**Issue by selfthinker** [splitbrain/dokuwiki#3445]
Some page titles and headings don't describe the page well enough. A good page title and main heading should "describe topic or purpose", describe both what the page is about (as well as the website it is in for page titles) and be unique.

The page title is the first thing that is read out to screen reader users and helps them understand what the page is about. It is often the first words someone reads about the page when they encounter it in search engine results or when they are bookmarked. A good page title also helps people with cognitive impairments to understand better what a page is about. It is better for usability as well.

The following either fail [2.4.2 Page Titled](https://www.w3.org/TR/WCAG21/#page-titled) or [2.4.6 Headings and Labels](https://www.w3.org/TR/WCAG21/#headings-and-labels).

## WCAG fails

* [ ] The page title of the [search results](https://grimm.haz.wiki/start?do=search&id=start&q=king) is always "Search [{Wiki Title}]" irrespective of what was searched for and is not unique.
That fails 2.4.2 and has a medium priority.
A way to fix it is to include the search term (and probably applied search filters).

* [ ] Similarly, the main heading of the search results is always "Search" but should be more specific and unique.
That fails 2.4.6 and has a medium priority.
The same fix as to the page title applies, i.e. including the search term.

* [ ] The "External", "Internal" and "Interwiki" headings on the [syntax page](https://grimm.haz.wiki/wiki:syntax) should be clearer when read out of context.
That fails 2.4.6 and has a low priority.
One fix is to add the word "Links", i.e. "Internal Links" instead of "Internal", etc.

## Other non-fail accessibility issues

These don't strictly fail WCAG but are still accessibility issues. Fixing them is recommended but optional for passing WCAG.

* [ ] The page title of the syntax page is "wiki:syntax [{Wiki Title}]". Although that gives enough of an idea what the page is about, it could be clearer.
One possible fix is using the [useheading config option](https://www.dokuwiki.org/config:useheading). It could be documented somewhere which options are good for accessibility and/or mentioned on that config page.

* [ ] The page title of the [edit view](https://grimm.haz.wiki/animals?do=edit) is "✎ {page name} [{Wiki Title}]'). It should be clearer that this is the edit view of the page. The pencil emoji is theoretically programmatically determinable but is neither read out by NVDA nor by VoiceOver (probably because the character is not "recommended for general interchange").
One fix is to use a better supported emoji, like ✏️. But saying it in words ("Editing...") is even better for screen reader users and users with cognitive impairments.
